fixtures-processor/src/main/kotlin/com/theblueground/fixtures/ProcessedParameterMapper.kt

```diff
@@ -20,19 +21,18 @@ internal class ProcessedParameterMapper(
 
     fun mapParameter(parameterValue: KSValueParameter): ProcessedFixtureParameter {
         val resolvedType = parameterValue.type.resolve()
-        return mapParameter(
-            parameterValue = parameterValue,
-            parameterType = resolvedType,
-            parameterClassDeclaration = (resolvedType.declaration as KSClassDeclaration),
-        )
+        return when (val declaration = resolvedType.declaration) {
+            is KSTypeAlias -> mapParameter(parameterValue, declaration.type.resolve())
```

Comment:
Thank you for your contribution! Can we also add a test for this case?

Reply:
Test added